chapters/101/index.md

```diff
@@ -1,167 +1,113 @@
 
+Every Pea runs inside a Pod. Sometimes multiple copies of a Pea run in a single Pod to improve efficiency and scaling.
 
 
 ## Pod
 
+![Pod Cartoon](images/pod.svg)
 
-<span style="display: inline-block; width: 30%;">
-
-![Jina 101 Pod, Copyright by Jina AI Limited](./img/ILLUS8.png)
+A Pod is a container and interface for one or multiple Peas that have the same properties. It coordinates Peas to improve efficiency and scaling. Beyond that, a Pod adds further control, scheduling, and context management to its Peas.
 
-</span>
-
-<span style="display: inline-block; width: 30%; float:right">
-
-![Jina 101 Pod Remote, Copyright by Jina AI Limited](./img/ILLUS9.png)
-
-</span>
-
-
-**Pod is a group of Peas with the same property.** Peas are running in parallel inside a Pod. Pod unifies the network interfaces of those Peas, making them look like one single Pea from outside. Beyond that, a Pod adds more control, scheduling and context management to the Peas.
-
-Pod can be run either on local host or on different computers over a network. 
-
-
-
-
-## Flow
-
-
-
-<span style="display: inline-block; width: 30%;">
-
-![Jina 101 Flow, Copyright by Jina AI Limited](./img/ILLUS10.png)
-
-</span>
-
-
-
-**Flow represents a high-level task**, e.g. indexing, searching, training. It manages the states and context of a group of Pods, orchestrating them to accomplish one task. Flow embraces diversity, whether a Pod is remote or in the Docker container, one Flow to rule them all!
-
-
-
-
-## From Micro to Macro
-
-
-Jina is a happy family. You can feel the harmony when you use Jina. 
-
-You can design at the micro-level and scale that up to the macro-level. YAMLs becomes algorithms, threads become processes, pods become flows. The patterns and logic always remain the same. This is the beauty of Jina. 
-
-<span style="display: inline-block; width: 80%;">
-
-![Jina 101 All Characters, Copyright by Jina AI Limited](./img/ILLUS11.png)
-
-</span>
 
+## Driver
 
-<p align="center">
-    ✨<b>Intrigued? Try our "Hello, World!" and build your neural image search in 1 min. </b>
-</p>
+![Driver Cartoon](images/driver.svg)
 
-<p align="center">
-    ✨<b>Unleash your curiosity and happy searching! </b>🔍
-</p>
+A Driver “translates” input and output messages for an Executor. Each Executor requires a different data format to perform its task. Therefore, a Driver interprets incoming messages into Documents and extracts required fields for an Executor. 
 
+## What comes next?
 
-> Checkout [our Github for multi-lingual version of this 101 Guide](https://github.com/jina-ai/jina/blob/master/docs/chapters/101/).
+Jina is a happy family. You can feel the harmony when you use Jina.
 
+You can design at the micro-level and scale up to the macro-level. YAMLs becomes algorithms, threads become processes, Pods become Flows. The patterns and logic always remain the same. This is the beauty of Jina.
```

Comment:
why do we say threads become processes?

Reply:
@JoanFM this part was from the original 101. We should probably remove it if this is incorrect?

Reply:
@alexcg1 do you know the origin of `threads become processes` maybe?

Reply:
I have no idea about the origin. Likely written by @hanxiao ?

Also:
- `YAMLs` -> `YAML files` (or `YAML becomes` ok too)

Reply:
I just removed the potentially confusing part `threads become processes` and fixed the typo.